### ISSUE TITLE
$rootScope.$digest() causing error when dismissing ($apply already in progress)

### ISSUE THREAD

**Issue by tosh001** [sachinchoolur/angular-flash#27]
When I fire Flash.clear() I get a $apply already in progress error.

If I comment out $rootScope.$digest() the error goes away

Any thoughts?

**Comment by holtalanm** [comment]
better fix than commenting out digest would be to write a new function that excludes it for clearing.

I think you still want the $digest in there when you dismiss, but I'm not sure.


I ran into this exact issue earlier.  I'll submit a pull request here in a few with my fix for it for review.

**Comment by RoopeHakulinen** [comment]
Thank you @tosh001 for reporting this, and @holtalanm for your PR. I actually thought I fixed this already but apparently I published version 2.2.6 without any actual code change. I must have accidentally changed the dist files and then overwritten them by building src folder again. 

@holtalanm's fix would probably do the work too, but I don't actually see any need for the _$digest_ at that place so I fixed this straight to master by removing the whole _$digest_ call.
